app/frontend/Settings/About/Debug.svelte

```diff
@@ -0,0 +1,53 @@
+<vbox>
+  <h1>Debug</h1>
+  <button on:click={() => logHistory.clear()}>Clear log history</button>
```

Comment:
Move to bottom, please

Also, please add a button to copy all contents to clipboard (we have a few other places where we copy to clipboard). This "Copy to clipboard" button is above the "Clear log history" button.